cypress/integration/search-page.spec.js

```diff
@@ -0,0 +1,13 @@
+import carMachine from '../../machines/car-machine';
+
+describe('Test search page', () => {
+  it('should be visible', () => {
+    cy.visit('/search');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bandvov/first-nextjs/pullRequest?prid=7125704)
```suggestion
    cy.visit("/search");
```